src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -68,48 +68,54 @@ <h4 class="panel-title border-bottom-small" translate>
         <h4 class="panel-title border-bottom-small" translate>
           {{'GIFT_AMOUNT'}}
         </h4>
-        <div class="panel-body pt0">
+        <div class="panel-body p0">
           <div ng-if="$ctrl.useSuggestedAmounts">
-            <div class="radio radio-custom-amount"
-                 ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
-              <label>
-                <input name="suggestedAmount"
-                       type="radio"
-                       ng-click="$ctrl.changeAmount(suggested.amount)"
-                       ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
-                {{$ctrl.suggestedAmount(suggested.amount)}} {{suggested.label}}
-              </label>
-            </div>
-            <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
-              <div class="form-group">
+            <div class="button-group">
+              <div 
+                  role="button"
```

Comment:
Why did you move away from an actual button element?

Reply:
I think a styled radio input would work fine, but I'm not fully read-in on this task, so I've assumed there was some reason to switch from radio inputs to buttons.